packages/manager/apps/dedicated/client/app/models/BillingService.class.js

```diff
@@ -46,7 +46,11 @@ export default class BillingService {
   }
 
   hasAutomaticRenew() {
-    return this.renew.automatic;
+    return (
+      this.renew.automatic ||
```

Comment:
I am wondering, you can have renew.automatic === false and still have automatic renew activated?

Reply:
Currently waiting an API response about that

Reply:
Since this ticket has been validated then I process to the merge.
Feel free to open a new Pull Request if you have updates to apply.